### PR TITLE
Add tag with JIRA ticket to 3.2.9 test coverage

### PR DESCRIPTION
**Pull request by michalvavrik**
### Summary

Add tags for:

- TD: https://github.com/quarkus-qe/quarkus-test-suite/pull/1508 Ticket: https://issues.redhat.com/browse/QUARKUS-3685
- TD: https://github.com/quarkusio/quarkus/pull/36664 Ticket: https://issues.redhat.com/browse/QUARKUS-3672
- TD: https://github.com/quarkus-qe/quarkus-test-suite/pull/1515 Ticket: https://issues.redhat.com/browse/QUARKUS-3674
- TD: https://github.com/quarkus-qe/quarkus-test-suite/pull/1513 Ticket: https://issues.redhat.com/browse/QUARKUS-3680

Please select the relevant options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Dependency update
- [x] Refactoring
- [ ] Backport
- [ ] New scenario (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [x] Methods and classes used in PR scenarios are meaningful
- [x] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

File: http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/BaseHttpAdvancedReactiveIT.java [hunk 1]
```diff
@@ -430,6 +430,7 @@ public void constraintsExist() throws JsonProcessingException {
     }
 
     @Test
+    @Tag("QUARKUS-3672")
     @Tag("https://github.com/quarkusio/quarkus/pull/36664")
```
Comment (jedla97) [inline http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/BaseHttpAdvancedReactiveIT.java:434]: Maybe we can remove the tag with upstream issue as now Quarkus link it together.

**Review by jedla97 — COMMENTED** [review]
Look good just that one thing. It's not necessary I'm fine with both way but maybe your thoughts on this.

File: http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/BaseHttpAdvancedReactiveIT.java (hunk 1 above)
Comment (michalvavrik) [reply http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/BaseHttpAdvancedReactiveIT.java:434]: oki

**Review by jedla97 — APPROVED** [review]
Thanks for the update! Looks good!

**Pull request merged**